proposals/lip-0043.md

```diff
@@ -209,64 +205,63 @@ Let `trs` be a transaction with module ID `MODULE_ID_INTEROPERABILITY` and comma
-    * The following [registration CCM][regCCM] is added to `interopSidechainAcc.outbox` by calling the `addToOutbox(chainID, registrationCCM)` logic as specified in Introduce Interoperability module LIP:
+  * `storeValue`: `sidechainAccount` where `sidechainAccount` is the chain account object, serialized according to the `chainAccountSchema` schema defined in Introduce Interoperability module LIP, and initialized as:
+    * The `sidechainAccount.name` property is equal to the `trs.params.name` property.
+    * The `sidechainAccount.networkID` property is calculated as the output of `SHA-256(trs.params.genesisBlockId || senderAddress)` where `||` indicates bytes concatenation and `senderAddress` is the address of the user account corresponding to the `trs.senderPublicKey` property.
+    * The `sidechainAccount.activeValidators` property is equal to the `trs.params.initValidators` property.
+    * The `sidechainAccount.certificateThreshold` property is equal to the `trs.params.certificateThreshold` property.
+    * The following [registration CCM][regCCM] is added to `sidechainAccount.outbox` by calling the `addToOutbox(chainID, registrationCCM)` logic as specified in Introduce Interoperability module LIP:
 
     ```java
     registrationCCM = {
-      "moduleID": MODULE_ID_INTEROPERABILITY,
-    	"crossChainCommandID": CROSS_CHAIN_COMMAND_ID_REGISTRATION,
-    	"index": 0,
-    	"sendingChainID": MAINCHAIN_ID,
-    	"receivingChainID": chainID,
-    	"fee": 0,
-    	"status": 0,
-    	"params": {
-    		"networkID": SHA-256(trs.params.genesisBlockId || senderAddress),
-    		"name": trs.params.name,
-    	}
+        "moduleID": MODULE_ID_INTEROPERABILITY,
+        "crossChainCommandID": CROSS_CHAIN_COMMAND_ID_REGISTRATION,
+        "nonce": 0,
+        "sendingChainID": MAINCHAIN_ID,
+        "receivingChainID": chainID,
+        "fee": 0,
+        "status": 0,
+        "params": {
+            "networkID": SHA-256(trs.params.genesisBlockId || senderAddress),
+            "name": trs.params.name,
+        }
     }
     ```
 
     * The rest of the properties, as specified in Interoperability module LIP, are initialized to their default values.
-* An outbox root store as specified in Introduce Interoperability module LIP is created as:
+* An entry in the outbox root substore as specified in Introduce Interoperability module LIP is created as:
   * `storePrefix`: `STORE_PREFIX_OUTBOX_ROOT`.
   * `storeKey`: `uint32be(chainID)`.
-  * `storeValue`: The property `interopSidechainAcc.outbox.root` serialized according to the `outboxRoot` schema defined in Introduce Interoperability module LIP.
-* A name store as specified in Introduce Interoperability module LIP is created as:
+  * `storeValue`: The property `sidechainAccount.outbox.root` serialized according to the `outboxRootSchema` schema defined in Introduce Interoperability module LIP.
```

Comment:
```suggestion
  * `storeValue`: The property `sidechainAccount.outbox.root` serialized according to the `outboxRootSchema` schema defined in "Introduce Interoperability module" LIP.
```

Reply:
and also later in the text

Reply:
the quotes are also missing once before as well

Reply:
there are many occurrences of this LIP reference without quotes. I would not add them since we are going to update the LIP title soon. And it is currently clear enough as it is (Also, I think I have not used quotes in similar LIP references in other LIPs

Reply:
cbr